libs/mocks/build.gradle

```diff
@@ -21,7 +21,3 @@ allprojects {
         
     }
 }
-
-task clean(type: Delete) {
-    delete rootProject.buildDir
-}
```

Comment:
This was causing the scripts to fail and it deletes the `build/` directory where the outputs are placed.